core/crop/structure.py

```diff
@@ -42,24 +42,26 @@
     ID_COL_NAME,
 )
 
+from sqlalchemy.ext.declarative import declarative_base
 
 db = SQLAlchemy()
 BASE = db.Model
 
+#BASE = declarative_base()
```

Comment:
We do not need this anymore. Base is created using:

db = SQLAlchemy()
BASE = db.Model